sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -127,7 +127,7 @@ public async Task TablesTeardown()
         /// <param name="partitionKeyValue">The partition key to create for the entity.</param>
         /// <param name="count">The number of entities to create</param>
         /// <returns></returns>
-        protected static List<TableEntity> CreateTableEntities(string partitionKeyValue, int count)
+        internal static List<TableEntity> CreateTableEntities(string partitionKeyValue, int count)
```

Comment:
Should we consider refactoring these into a stand-alone utility class if they're being used outside of derived classes?